<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storage</class>
 <widget class="QWidget" name="storage">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/qt 图标/新增.jpg</normaloff>:/qt 图标/新增.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 75 12pt &quot;楷体&quot;;
background-color: rgb(144, 144, 144);
color:rgb(0, 0, 0)
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_13">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>生成厂商</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>标签编号</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_11">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>单位</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>订 单 号</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品编号</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品数量</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="10" column="4" colspan="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>管 理 员</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品类别</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_12">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>生成日期</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_2">
     <property name="minimumSize">
      <size>
       <width>145</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>货物入库登记</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>仓库编号</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>入库时间</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::NoEcho</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>NOW DATE</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>产品名称</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
